packages/components/src/templates/next/components/internal/BlogCard/BlogCard.stories.tsx

```diff
@@ -100,7 +100,7 @@ export const TagsWithoutImage: Story = {
   args: generateArgs({
     title: "Collection card without tags",
     withoutImage: true,
-    description: "This is a random description that will be on the card",
+    description: "This is a random description\nthat will be on the card",
```

Comment:
nit - should we make a new story for this? since we are testing for different things (one is tag without image, another is newline)

Reply:
Hmm I didn't create a new story cos it isn't really a standalone feature, plus it doesn't conflict with the other functionality of the card.

Reply:
oh i was thinking it might be misleading to lump them together under `TagsWithoutImage` because we should be treating stories like actual test cases and thus following the principle of isolating every test case to testing only one thing

Reply:
Yup agreed if it is for a specific feature, but in this case it is like any other description which I thought was not sufficient to have its own stories.

Also I'm not sure when did we start to treat stories like test cases, but I think visual tests should be kept separate from Storybook. These stories are more for simulating instances which are difficult to get to (e.g. loading states) to facilitate review by design, whereas visual tests say using Playwright are more appropriate to perform the testing that you mentioned.

Reply:
Thanks for sharing - i see where you are coming from and agree!

Though since we’re not quite using Playwright for visual testing (at the moment) + this approach is using Storybook to some extent as a testing proxy, it might be clearer to split them up so that it's easier to understand and maintain in the future.

That said, i’m not blocking this PR - just thought to share that this approach might feel a bit conflated and might introduce some potential for confusion. Let's proceed to merge this and we can maybe discuss this again later on